drivers/vulkan/rendering_device_vulkan.h

```diff
@@ -43,7 +43,7 @@
 #define _DEBUG
 #endif
 #endif
-#include "vk_mem_alloc.h"
+#include "thirdparty/vulkan/vk_mem_alloc.h"
```

Comment:
Does this change fix something specific?

`driver/vulkan/SCsub` already adds `thirdparty/vulkan` in the include path so it doesn't need to be specified explicitly (same as `volk.h` and `vulkan/vulkan.h` below). This could use `<vk_mem_alloc.h>` instead of `"vk_mem_alloc.h"` to be consistent though.